go-controller/pkg/nbdb/gen.go

```diff
@@ -0,0 +1,2 @@
+//go:generate modelgen -p nbdb -o . ovn-nb.ovsschema
```

Comment:
Can you add a make directive that installs `modelgen`? Or somehow vendor it? I can show you a dependencymagnet hack if that would help.

Reply:
it's easy enough to add a  `go install github.com/ovn-org/libovsdb/cmd/modelgen` to the Makefile.
It's only every needed to run `make generate`

Reply:
we should pin it to a specific version

Reply:
what about something like?

```diff
--- a/go-controller/hack/update-codegen.sh
+++ b/go-controller/hack/update-codegen.sh
@@ -4,6 +4,22 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# generate ovsdb bindings
+if  ! ( command -v modelgen > /dev/null ); then
+  echo "controller-gen not found, installing sigs.k8s.io/controller-tools"
+  olddir="${PWD}"
+  builddir="$(mktemp -d)"
+  cd "${builddir}"
+  GO111MODULE=on go get -u github.com/ovn-org/libovsdb/cmd/modelgen
+  cd "${olddir}"
+  if [[ "${builddir}" == /tmp/* ]]; then #paranoia
+      rm -rf "${builddir}"
+  fi
+fi
+
+go generate ./pkg/nbdb
+go generate ./pkg/sbdb
```